firmware/version_data.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
```

Comment:
Can you add [a docstring](https://www.python.org/dev/peps/pep-0257/) for this file?